styles.css

```diff
@@ -0,0 +1,125 @@
+
+.mobile-header {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0.1px;
+  top: 0.1px;
```

Comment:
- [ ] Please delete ALL the `left`, `top`, `right` and `bottom` references that you have in your document, you could use `padding` and `margin` properties instead